test/chrome_test.js

```diff
@@ -47,9 +48,25 @@ describe("StorageArea", function() {
 
   /** @test {StorageArea#constructor} */
   describe("#constructor", function() {
-    it("should expose endpoint and areaName", function() {
+    it("should expose areaName", function() {
       expect(area.areaName).to.eql("sync");
-      expect(area.endpoint).to.eql("http://localhost:8080/v1/");
+    });
+  });
+
+  describe(".config setter", function() {
+    it("should set the config", function() {
+      const area = new StorageArea("sync");
+      area.config = { type: "kinto", remote: "http://localhost:8080/v1/" };
+      expect(area.config.remote).to.eql("http://localhost:8080/v1/");
+    });
+
+    it("should configure a sync timer", function(done) {
+      const area = new StorageArea("sync");
+      area.config = { type: "kinto", interval: 123000 };
+      setTimeout(function() {
```

Comment:
This is not really a timer, it's just waiting for https://github.com/michielbdejong/kinto-chrome/blob/chrome-storage-sync-config/lib/chrome.js#L89-L92 to resolve its promise. I changed it to `Promise.resolve().then` to make this clearer, and added a TODO comment which I plan to take care of in #12.